### DISCUSSION TITLE
Deployment of partcular project in nested level on github pages

### DISCUSSION THREAD

**Discussion by akhilcharugulla** [community/community#135937] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

I have multiple projects inside frontend projects. Among them I would like to host my [Rockpapaerscissor](https://github.com/akhilcharugulla/frontend-projects/tree/master/Rockpapaerscissor)project.
I tried to deploy clicking settings, selecting branches as master, selected /docs (I am not sure what to select here in this case)
When to select root and when to select /docs I don't understand

Can someone please explain , with an example.

![image](https://github.com/user-attachments/assets/d5869c5f-b13f-431c-8539-1bf3b93f2045)

**Comment by akhilcharugulla** [comment]
Currently I get build failure which is expected:
https://github.com/akhilcharugulla/frontend-projects/actions/runs/10411152559/job/28834515052

**Comment by davevad93** [comment]
Hi @akhilcharugulla ,

You have to select `/root` when your project files are located in the root of your repo, for example:

`/repo`
`├── RockPaper.css`
`├── RockPaper.js`
`└── index.html`

On the other hand you have to select `/docs` if your files are inside the docs folder, for example:

`/repo`
`└── /docs`
`      ├── RockPaper.css`
`      ├── RockPaper.js`
`      └── index.html`

I suggest you to create a new repo with the `RockPaperScissors` project and deploy it.

**Comment by akhilcharugulla** [comment]
Hi @davevad93  Thanks for the response
I will create new repository. Is there any chance here with this folder structure?

**Reply by davevad93** [reply, marked as answer]
Hey @akhilcharugulla with the current folder structure no, create a new repo and put `RockPaper.css`, `RockPaper.js` and `index.html` in the root, then go to `Settings` => `Pages`, and select `master` `/root`. Your folder structure should be as follows:

`/repo`
`├── RockPaper.css`
`├── RockPaper.js`
`└── index.html`
